dev/wlp.lib.extract/src/wlp/lib/extract/SelfExtractRun.java

```diff
@@ -339,6 +339,8 @@ private static int runServerInline(String extractDirectory,
 
         Properties props = System.getProperties();
 
+        // Setting this tells us the boostrap to ignore any WLP_USER_DIR that happens to be set and use the <install>/usr from the extracted server install.
+        props.setProperty("wlp.prefer.default.user.dir", "true");
```

Comment:
Is there a case where we would want some other directory in a self extracted run? This locks that in - which may be ok in this scenario but wondering if there are cases where it might make sense to have some other dir even in self runable jar?

Reply:
@NottyCode  any opinions here?